ci-operator/jobs/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master-presubmits.yaml

```diff
@@ -14,7 +14,6 @@ presubmits:
       - args:
         - --artifact-dir=$(ARTIFACTS)
         - --target=[images]
-        - --target=[release:latest]
```

Comment:
@smarterclayton you MUST fix this. ASAP.

Reply:
Every [images] target in a presubmit that is part of the openshift release should have this target added.

Reply:
If you generate this for every job that has tag_spec for now that is sufficient.

Reply:
So can we just add this to `[images]` presubmit, just like we do `--promote` to postsubmits? Should postsubmits  have the `[release:latest]`?